fe/fe-core/src/test/java/com/starrocks/http/TransactionLoadActionTest.java

```diff
@@ -0,0 +1,311 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.